api/types/types.go

```diff
@@ -35,3 +35,9 @@ type ContainerWaitResponse struct {
 type ContainerCommitResponse struct {
 	ID string `json:"Id"`
 }
+
+// GET /containers/.../changes
```

Comment:
I suggest `/containers/{name}/changes`. Note to maintainers: we should probably standardize the comments in this file.